homeassistant/components/script/blueprints/random_color.yaml

```diff
@@ -0,0 +1,109 @@
+blueprint:
+  name: Random Color
```

Comment:
Should the name be more descriptive so a new user is interested in looking at it?